backend/src/database/migrations/V1764931554__create_dashboard_metrics_sink.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE public."dashboardMetricsTotalSnapshot" (
+    id TEXT PRIMARY KEY DEFAULT 'snapshot',
```

Comment:
If primary key is text I'd prefer it to be at least varchar with a specified text size limitation so that db can optimize for it.